docker/staging/docker-compose-db.yml

```
@@ -0,0 +1,4 @@
+include:
+  - networks.yml
+  - service-postgres.yml
+  - service-redis.yml
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)